MIGRATION_GUIDE.TXT

```diff
@@ -25,6 +25,15 @@ MIGRATION GUIDE FROM GDAL 3.6 to GDAL 3.7
   drivers with write support are encouraged to implement it for error
   propagation.
 
+- Pansharpening now requires that panchromatic and multispectral bands have
+  valid geotransform (in early versions, it was assumed in the case of missing
+  geotransform that they covered the same geospatial extent).
```

Comment:
```suggestion
  valid geotransform (in early versions the geotransform was not considered, i.e. it was expected that all bands covered the same geospatial extent).
```

Reply:
your suggestion is actually not quite true. If the geotransforms of the pan and ms band differed by more than one pixel of the pan, then the SpatialExtentAdjustment=Union default logic would add the necessary padding so their geotransform was taken into account (but with integer rounding, so not a sub-pixel accuracy from what I can see)